images/installer.yml.in

```diff
@@ -0,0 +1,18 @@
+kernel:
+  image: KERNEL_TAG
+  cmdline: "rootdelay=3 root=/dev/sdb"
```

Comment:
Can we avoid hardcoding /dev/sdb? I'm thinking about future hardware platforms where it might appear as some other disk.

Reply:
Yes, We could create a UUID for the installer partition. I can do it in another changeset, as it can get a bit messy. I will definitely add that once we add more platforms.